### ISSUE TITLE
Paginate MissingMethodException

### ISSUE THREAD

**Issue by Noxxys** [JohnnyCrazy/SpotifyAPI-NET#561]
Hi.
While using the `Paginate` method as described on your website, I get a runtime error:
```
  System.MissingMethodException -  Method not found: 'System.Collections.Generic.IAsyncEnumerable`1<!!0> SpotifyAPI.Web.SpotifyClient.Paginate(SpotifyAPI.Web.IPaginatable`1<!!0>, SpotifyAPI.Web.IPaginator, System.Threading.CancellationToken)'
```
I'm using version 6.0 of your library in a C# class library with .NET Standard 2.1, which is referenced by a WPF application with .NET 5.0. `PaginateAll` is working fine.

```c#
var sourcePlaylist = await spotifyClient.Playlists.Get(sourcePlaylistId).ConfigureAwait(false);
var sourceUris = new List<string>();

// This doesn't work
await foreach (var playlistTrack in spotifyClient.Paginate(sourcePlaylist.Tracks))
{
    var track = (FullTrack)playlistTrack.Track;
    sourceUris.Add(track.Uri);
}

// but this works
//var tracks = await spotifyClient.PaginateAll(sourcePlaylist.Tracks).ConfigureAwait(false);

//sourceUris = tracks
//    .Select(t => ((FullTrack)t.Track).Uri)
//    .ToList();
```

**Comment by JohnnyCrazy** [comment]
Could you create a minimal reproducible example and upload it to github? `IAsyncEnumerable` should be available in .NET 5.0 and .NET Standard 2.1, so the only possible error I can think of could be a missing reference import.

**Comment by Noxxys** [comment]
Hi Jonas. I've created the repo: https://github.com/Noxxys/SpotifyIssue
I found another problem while comparing with a .NET 5.0 class library, please see the readme file.

**Comment by JohnnyCrazy** [comment]
Thanks for the repro repo, was a great help 👍

The `master` contains a fix for this issue. I was using `#if NET_STANDARD2_1` checks to enable the `Paginate` method. This was not very future-friendly (see `NET5`), so I replaced them with `#if !NET_STANDARD2_0` checks.

I will also release a new version containing this fix soon! Thanks again :)

**Comment by JohnnyCrazy** [comment]
Fixed by 81245b2

**Comment by Noxxys** [comment]
thank you for the fix
